scripts/wdio/wdio-runner.js

```diff
@@ -39,11 +52,11 @@ async function wdioRunner(options) {
         .run()
         .then(
           (code) => {
-            if (code === 1 && !continueOnFail) {
+            if (code === 1 && !continueOnFail && !exitProcess) {
```

Comment:
I think it's better to have an inline comment that explains this if-else block for future debugging (if necessary).